src/plugins/legacy/itkDataImage/writers/itkNiftiDataImageWriter.h

```diff
@@ -24,6 +24,7 @@ class ITKDATAIMAGEPLUGIN_EXPORT itkNiftiDataImageWriter: public itkDataImageWrit
     virtual QString identifier()  const;
     virtual QString description() const;
     virtual QStringList handled() const;
+    virtual QStringList supportedFileExtensions() const;
```

Comment:
~~virtual~~ QStringList supportedFileExtensions() const **override**;

Reply:
Do I make the change for the others functions since i am changing this file?

Reply:
If you have time it's better :)